arquillian-environment-reporter/arquillian-environment-reporter-impl/src/main/resources/org.arquillian.environment.reporter.impl.ArquillianEnvironmentKeys.properties

```diff
@@ -0,0 +1,7 @@
+ENVIRONMENT_SECTION_NAME.value=Environment
+JAVA_VERSION.value=Java Version
+TEST_RUNNER.value="Test Runner"
+DOCKER.value="Inside Docker"
+TIMEZONE.value="TimeZone"
+CHARSET.value="Default Charset"
+OPERATIVE_SYSTEM.value="Operative System"
```

Comment:
Some short description would be helpful in some cases. TEST_RUNNER and CHARSET in particular. When we have the opportunity to add a description there, please do it. At least in this core project.